bin/translation-status.js

```diff
@@ -29,9 +29,9 @@ glob(LOCALES_PATH, (err, files) => {
       languageCode,
       totalStrings,
       translatedStrings,
-      missingStrings: totalStrings-translatedStrings
+      missingStrings: totalStrings - translatedStrings
     }
   })
 
-  console.log(translationStatus)
+  console.log(JSON.stringify(translationStatus))
```

Comment:
This could probably be pretty printed instead, `JSON.stringify(translationStatus, null, 2)` but I have no strong feelings for this. It's probably easier to parse from other scripts...